repair/repair.cc

```diff
@@ -119,6 +119,16 @@ std::ostream& operator<<(std::ostream& out, row_level_diff_detect_algorithm algo
     return out << "unknown";
 }
 
+static size_t get_nr_tables(const replica::database& db, const sstring& keyspace) {
+    size_t ret = 0;
+    for (auto &&e : db.get_column_families_mapping()) {
+        if (e.first.first == keyspace) {
+            ret++;
+        }
+    }
+    return ret;
```

Comment:
perhaps it would be more idiomatic to rewrite it using std::count_if or boost::count_if

Reply:
I would avoid boost if possible. 

I am using std::count_if in v2.

Thanks.